features/step_definitions/admin.rb

```diff
@@ -22,3 +22,22 @@
     raise "cannot create secret, see log"
   end
 end
+
+Given /^I save the project name hosting #{QUOTED} resource named #{QUOTED} to#{OPT_QUOTED} clipboard/ do |resource, res_name, cb_name|
+  ensure_admin_tagged
+
+  cb_name ||= :namespace
+  @result = admin.cli_exec(:get, all_namespaces: true, resource: resource, o: 'yaml')
+  if @result[:parsed].nil?
+    cb[cb_name] = nil
```

Comment:
this looks like a big issue and we better fail. At least output a warning like on line 36